ipam/api.go

```diff
@@ -82,16 +82,6 @@ type ReleaseAddressRequest struct {
 	Address string
 }
 
-// ErrorResponse is a formatted error message that libnetwork can understand
-type ErrorResponse struct {
-	Err string
-}
-
-// NewErrorResponse creates an ErrorResponse with the provided message
-func NewErrorResponse(msg string) *ErrorResponse {
-	return &ErrorResponse{Err: msg}
-}
-
```

Comment:
This is common to all drivers (network and IPAM). No need to duplicate this structure definition. I'm moving this to `sdk`.